test/homework_01/test_task_02.py

```diff
@@ -21,6 +46,16 @@ def test_positive_0_1():
     assert check_fibonacci([0, 1])
 
 
+def test_positive_0_1_1():
+    """Testing that [0, 1, 1]"""
```

Comment:
that what?

Please remove docstrings from tests and move their content into the test names.